### PR TITLE
feat(oma-aqm): Added the OMA Alert Quality Management dashboard

### PR DESCRIPTION
**Pull request by AlecIsaacson**
# Summary

This PR adds support for the OMA Alert Quality Management dashboard, as well as pointers to the docs required to configure the webhook the dashboard requires.

## Pre merge checklist

- [ X] Did you check you NRQL syntax? - Does it work?
- [ X] Did you check your dashboard image quality? -  Do they look good?
- [ N/A] Did you check that your alerts actually work?
- [ X] Did you include an InstallPlan and Documentation reference?
- [ X] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [ X] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

Attach images of any visual changes, such as a dashboard here.
alert-quality-management

### CODE REVIEW

File: quickstarts/oma-aqm/config.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+name: alert-quality-management
+
+# Displayed in the UI (required)
+title: Alert Quality Management Quickstart
+
+# Long-form description of the quickstart (required)
+description: |
+  Alert Quality Management (AQM) focuses on reducing the number of nuisance incidents so that you focus only on alerts with true business impact.  This reduces alert fatigue and ensures that you and your team focus your attention on the right places at the right times.
+  
+  For more information, see https://docs.newrelic.com/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/aqm-implementation-guide/
```
Comment (jpvajda) [inline quickstarts/oma-aqm/config.yml:11]:
```suggestion
  For more information view the [implementation guide](https://docs.newrelic.com/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/aqm-implementation-guide/).
```

File: quickstarts/oma-aqm/config.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Reduce the volume of nuisance incidents with the Alert Quality Management dashboard and process.
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
```
Comment (jpvajda) [inline quickstarts/oma-aqm/config.yml:18]: by setting this to community, it means it's not an official New Relic quickstart and was built by 3rd party creators, I just want to make sure is the correct level to set this to.

https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#levels

File: quickstarts/oma-aqm/config.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+# Authors of the quickstart (required)
+authors:
+  - Alec Isaacson (New Relic)
+  - Andrew Faria (New Relic)
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - oma
```
Comment (jpvajda) [inline quickstarts/oma-aqm/config.yml:27]: what is oma specifically?

File: quickstarts/oma-aqm/config.yml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  - observabilty maturity
```
Comment (jpvajda) [inline quickstarts/oma-aqm/config.yml:28]: is this an actual term that is worthy of a dedicated keyword?

File: quickstarts/oma-aqm/config.yml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  - AQM
+  - alert quality management
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+installPlans:
+  - oma-aqm
```
Comment (jpvajda) [inline quickstarts/oma-aqm/config.yml:35]: I didn't see any actual install plan in this directory for oma-aqm

https://github.com/newrelic/newrelic-quickstarts/tree/main/install

File: quickstarts/oma-aqm/config.yml [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+documentation:
+  - name: Installation Docs
+    url: https://docs.newrelic.com/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/aqm-implementation-guide/
```
Comment (jpvajda) [inline quickstarts/oma-aqm/config.yml:39]:
```suggestion
    url: >- 
      https://docs.newrelic.com/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/aqm-implementation-guide/
```

File: quickstarts/oma-aqm/config.yml [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+    description: The implementation guide will show you how to implement the AQM process, including the webhook required to generate the data that feeds this dashboard.
```
Comment (jpvajda) [inline quickstarts/oma-aqm/config.yml:40]:
```suggestion
    description:  | 
        The implementation guide will show you how to implement the AQM process, including the webhook required  to generate the data that feeds this dashboard.
```

**Review by jpvajda — CHANGES_REQUESTED** [review]
I made several comments of feedback I'm also looking into why there is such this PR has a package-lock.json file and change to yarn.lock.

File: quickstarts/oma-aqm/config.yml (hunk 5 above)
Comment (jpvajda) [reply quickstarts/oma-aqm/config.yml:35]: You can read more about adding an install plan here: https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#installplans

File: node_modules/.yarn-integrity [hunk 8]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (nr-kkenney) [inline node_modules/.yarn-integrity:1]: dont think we want to commit this.

File: utils/package-lock.json [hunk 9]
```diff
@@ -0,0 +1,7263 @@
+{
```
Comment (nr-kkenney) [inline utils/package-lock.json:1]: dont think we want to commit this, we have yarn.lock.

File: yarn.lock [hunk 10]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (nr-kkenney) [inline yarn.lock:1]: dont think we want to commit this also.

File: quickstarts/oma-aqm/config.yml (hunk 4 above)
Comment (AlecIsaacson) [reply quickstarts/oma-aqm/config.yml:28]: Yes.  OMA and "observability maturity" are terms that are part of an overall program created by NR to help our customers get better value from their observability solution.  See https://docs.newrelic.com/docs/new-relic-solutions/observability-maturity/introduction/ fro details

File: quickstarts/oma-aqm/config.yml (hunk 5 above)
Comment (AlecIsaacson) [reply quickstarts/oma-aqm/config.yml:35]: Removed installPlans: stanza.  The Fastly example that I followed also has an installPlans stanza that doesn't seem to point at an existing install plan.

File: quickstarts/oma-aqm/config.yml (hunk 3 above)
Comment (AlecIsaacson) [reply quickstarts/oma-aqm/config.yml:27]: OMA = Observability Maturity Architecture

File: node_modules/.yarn-integrity (hunk 8 above)
Comment (AlecIsaacson) [reply node_modules/.yarn-integrity:1]: Agreed.  The stock .gitignore includes utils/node_modules, so I'd expect it to not grab utils/node_modules/.yarn-integrity, but it clearly did.  

For now, I've removed .yarn-integrity from the repo.  Perhaps someone with better .gitignore experience can debug this and fix the .gitignore?

File: utils/package-lock.json (hunk 9 above)
Comment (AlecIsaacson) [reply utils/package-lock.json:1]: I think the package-lock.json file got created when I ran either sanitize-dashboards or validate_quickstarts so I could better understand why the unit test was rejecting my dashboard JSON.  I removed that file from the repo.  

I don't have enough experience with Node.JS to know the impact of adding package-lock.json to the .gitignore, which would be  a more elegant and long-lasting fix.

File: yarn.lock (hunk 10 above)
Comment (AlecIsaacson) [reply yarn.lock:1]: File removed.

Comment (AlecIsaacson) [reply yarn.lock:1]: File removed.

**Review by jpvajda — APPROVED** [review]

**Pull request merged**